fe1-web/model/network/method/message/Message.ts

```diff
@@ -71,4 +71,9 @@ export class Message {
       witness_signatures: (witnessSignatures === undefined) ? [] : witnessSignatures,
     });
   }
+
+  public toProtocolJson(): any {
```

Comment:
To keep type safety , we could define an interface = Message without messageData and return it here

Reply:
This of course will require some adaptations elsewhere